### PR TITLE
Feature/plugins per channel

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it adds support for configuring plugins per channel

# Impact

* [x] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/account/mutations/base.py
Comment (maarcingebala) [inline saleor/graphql/account/mutations/base.py:52]: The identical function is used in the `CheckoutCreate` mutation (and `clean_channel` as well). Try to extract these functions and reuse them. They could be added to `graphql/channel/utils.py` maybe.

**Comment by korycins** [comment]
FYI: We are waiting for small changes in the storefront before we will merge it.

File: saleor/graphql/account/mutations/account.py
Comment (maarcingebala) [inline saleor/graphql/account/mutations/account.py:110]: We could try to only pass the class name as a signle argument e.g. `error_class=AccountError` - the error codes should be the same in each case.

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**
